lib/properties.ts

```diff
@@ -95,7 +95,7 @@ export function parseProperties(blob: string, name: string): Record<string, Prop
     return props;
 }
 
-export function initialize(directory: string, hier) {
+export function initialize(directory: string, hier: string[]) {
     if (hier === null) throw new Error('Must supply a hierarchy array');
```

Comment:
wouldn't TS/Lint complain about `hier === null` being useless in this case?

Reply:
I think it should have, but it doesn't... Any `=== null` or `=== undefined` is allowed, even with `strictNullChecks=true` as we do.   I now tried to understand it better here: https://discord.com/channels/508357248330760243/1295283918949449810
I think the answers are decent.